test/lib/elements/types/ExportNamedDeclaration.js

```diff
@@ -75,6 +75,26 @@ describe('ExportNamedDeclaration', () => {
         expect(statement.source.value).to.equal('m');
     });
 
+    it('should accept a ExportDefaultSpecifier from another module - exportExtensions', () => {
+        var statement = parseAndGetStatement('export x from "m";');
```

Comment:
I think it's also good to have test like this: `export {x as y, z as w} from "m"` and `export {default as i} from "m"`